src/AHitbox.hpp

```diff
@@ -0,0 +1,23 @@
+/*
+** EPITECH PROJECT, 2024
+** OpenthisFuckinDoor
+** File description:
+** AHitbox
+*/
+
+#include "AEntity.hpp"
+
+class AHitbox:public AEntity
+{
+public:
+    AHitbox(float x, float y);
+    ~AHitbox() = default;
+    
+    std::pair<float, float> getHitboxSize() {return (_hitboxSize);}
+    void setHitboxSize(float x, float y) {_hitboxSize = std::pair<float, float>(x, y);}
```

Comment:
You can rename it `setSize` this the method is only accessable from an HitBox class

Reply:
no! absolutely not it will override getSize of entity

Reply:
my bad